docs/Contributing/API-for-contributors.md

```diff
@@ -595,7 +595,7 @@ Once base64-decoded, they are PEM-encoded certificate and keys.
 | team_id   | number | query | _Available in Fleet Premium_ The team ID to apply the custom settings to. Only one of `team_name`/`team_id` can be provided.          |
 | team_name | string | query | _Available in Fleet Premium_ The name of the team to apply the custom settings to. Only one of `team_name`/`team_id` can be provided. |
 | dry_run   | bool   | query | Validate the provided profiles and return any validation errors, but do not apply the changes.                                    |
-| profiles  | json   | body  | An array of strings, the base64-encoded .mobileconfig (macOS) or XML (Windows) files to apply.                                        |
+| profiles  | json   | body  | An array of objects, consisting of two properties. `profile` base64-encoded .mobileconfig (macOS) or XML (Windows) file and `labels` array of strings (label names).                                        |
```

Comment:
I think we forgot to update the docs, but this is actually an object:

```json
{
  "profile_name_1": "base-64-encoded-data",
  "profile_name_2": "base-64-encoded-data"
}

Reply:
there are two options:

```json
{
    "profile_name": {
        "profile": "base-64-string",
        "labels": ["foo"]
    }
}
```

or, consolidate everything into a single object:

```json
[
    {
        "profile": "base-64-string",
        "name": "profile_name",
        "labels": ["foo"]
    }
]
```

I'm going to start coding the second option but please shout if you want something different @marko-lisica @noahtalerman